docappendertest/docappendertest.go

```diff
@@ -85,6 +86,56 @@ func DecodeBulkRequest(r *http.Request) ([][]byte, esutil.BulkIndexerResponse) {
 	return indexed, result
 }
 
+// DecodeBulkRequestWithStats decodes a /_bulk request's body, returning the decoded documents
+// and a response body and stats about request.
+func DecodeBulkRequestWithStats(r *http.Request) (
```

Comment:
I suggest we use `DecodeBulkRequestWithStats` in `DecodeBulkRequest` and just discard the stats. This will avoid duplicating the code.

Reply:
Makes sense, addressed 👍